lib/Serialization/Serialization.cpp

```diff
@@ -2868,7 +2868,7 @@ void Serializer::writeDecl(const Decl *D) {
                             addDeclBaseNameRef(param->getArgumentName()),
                             addDeclBaseNameRef(param->getName()),
                             contextID,
-                            param->isLet(),
+                            (unsigned)param->getSpecifier(),
```

Comment:
Please use a separate enum for this, like the others in ModuleFormat.h. That way if we ever remove a specifier it won't affect the serialization.